src/wdio/visualcompare.js

```diff
@@ -0,0 +1,23 @@
+import path from 'path';
+import { LocalCompare } from 'wdio-visual-regression-service/compare';
+
+function getScreenshotName(ref) {
```

Comment:
this logic needs to be tweaked a little to add a few requirements @emilyrohrbough had